### PR TITLE
Get civil file content doc issue

### PR DESCRIPTION
**Pull request by KFloodBCGov**

### CODE REVIEW

File: src/libs/ords-pcss-client/ords.pcss.swagger.yaml [hunk 1]
```diff
@@ -96,7 +96,49 @@ paths:
...
+      tags:
+      - PCSS-CIVIL
+      security:
+      - basicAuth: []      
+      description: Get the civil file content document support data
+      responses:
+        '200':
+          description: output of the endpoint
+          schema:
+            type: object
```
Comment (alexjoybc) [inline src/libs/ords-pcss-client/ords.pcss.swagger.yaml:111]: ref must be at this level

File: src/libs/ords-pcss-client/ords.pcss.swagger.yaml [hunk 2]
```diff
@@ -96,7 +96,49 @@ paths:
...
+      tags:
+      - PCSS-CIVIL
+      security:
+      - basicAuth: []     
+      description: Get the civil file content document issue data
+      responses:
+        '200':
+          description: output of the endpoint
+          schema:
+            type: object
```
Comment (alexjoybc) [inline src/libs/ords-pcss-client/ords.pcss.swagger.yaml:132]: move ref here

File: src/libs/ords-pcss-client/ords.pcss.swagger.yaml (hunk 1 above)
Comment (KFloodBCGov) [reply src/libs/ords-pcss-client/ords.pcss.swagger.yaml]: Went through and did this to any that had the object: and properties: fields as well...

File: src/libs/ords-pcss-client/ords.pcss.swagger.yaml (hunk 2 above)
Comment (KFloodBCGov) [reply src/libs/ords-pcss-client/ords.pcss.swagger.yaml]: Went through and did this to any that had the object: and properties: fields as well...

**Pull request merged**